src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  color: #00acdc;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header__container {
+  padding: 0 50px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  flex-wrap: wrap;
+  gap: 60px;
+  max-width: 1200px;
+}
+
+.logo__img {
+  width: 100%;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+  list-style: none;
+  height: 60px;
+}
+
+.nav__link {
+  border-bottom: 4px solid transparent;
+  text-decoration: none;
+  transition:
+    color 0.5s,
+    border-color 0.5s;
+  color: black;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  border-bottom: 4px solid;
+}
+
+a.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
The class `is-active` is used to style active links. Ensure that this class is applied dynamically to the correct element in the HTML to reflect the active state accurately.